src/Functions.Worker.Extensions.OpenAI/Functions.Worker.Extensions.OpenAI.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <Description>Microsoft Azure Functions Worker OpenAI Extension</Description>
```

Comment:
should we mention for .NET isolated functions

Reply:
+1. I suggest "Azure Functions .NET Isolated worker extension for OpenAI"